### PR TITLE
GPDB6 Segment Installation Fix

### PR DESCRIPTION
**Pull request by chewygranola**
Implementing fix to allow GPDB6 segment install + multiple versions in one cluster via prefix directory.

### CODE REVIEW

File: datalab/helpers.go [hunk 1]
```diff
@@ -1,29 +1,52 @@
...
-	if len(strings.TrimSpace(v)) == 0 {
-		return true
+    
+        
+          
+    
+
+        
+    
+    @@ -18,7 +18,7 @@ func IsValueEmpty(v string) bool {
```
Comment (faisaltheparttimecoder) [inline datalab/helpers.go:22]: Seems like these are junk characters, can you fix this

File: datalab/helpers.go [hunk 2]
```diff
@@ -1,29 +1,52 @@
...
+    
+
+          
+          
+            
+    
 
+          
+    
+    @@ -93,8 +93,8 @@ func generateEnvByVagrantKey(command string) []string {
```
Comment (faisaltheparttimecoder) [inline datalab/helpers.go:47]: Seems like these are junk characters, can you fix this

**Comment by faisaltheparttimecoder** [comment]
Looks good to me, just some file correction that needs to sorted, will it be possible to fix those corrections ?

**Comment by chewygranola** [comment]
Whoops, looks like copy/paste went awry. Got another version and copied the file in directly.

**Pull request merged**